### PR TITLE
Pipeline support

### PR DESCRIPTION
**Pull request by shacharPash**

### CODE REVIEW

**Comment by shacharPash** [comment]
This is a temporary solution to support pipelines using MULTI and EXEC,
The next step is to change the code so that Redis stack commands can be called in this way (or similar):
```csharp
var pipeline = new Pipeline(db);
pipeline.Json().Set(...);
pipeline.Json().Get(...);
```
Without 
```csharp
.AddCommand(JsonCommandBuilder.
```

**Comment by chayim** [comment]
@shacharPash did you see the conflict?

**Comment by shacharPash** [comment]
> @shacharPash did you see the conflict?

I fixed it now 👍

File: tests/NRedisStack.Tests/Json/JsonTests.cs [hunk 1]
```diff
@@ -20,6 +20,22 @@ public void Dispose()
...
     }
 
+    [Fact]
+    public async Task TestJsonPipeline()
+    {
+        var pipeline = new Pipeline(ConnectionMultiplexer.Connect("localhost"));
+        pipeline.Db.ExecuteAsync("FLUSHALL");
+
+        string jsonPerson = JsonSerializer.Serialize(new Person { Name = "Shachar", Age = 23 });
+        var setResponse = pipeline.Json.SetAsync("key", "$", jsonPerson);
```
Comment (chayim) [inline tests/NRedisStack.Tests/Json/JsonTests.cs:30]: perfect

**Comment by Jeevananthan-23** [comment]
Hi @shacharPash, working on pipeline examples and found that **ModulePrefix** has not been added for some of the module commands except `JSON`
```cs
public class Pipeline
{
    public Pipeline(IConnectionMultiplexer muxer)
    {
        _batch = muxer.GetDatabase().CreateBatch();
    }

    private IBatch _batch;

    public void Execute()
    {
        _batch.Execute();
    }

    public IJsonCommandsAsync Json => new JsonCommandsAsync(_batch); // Json

   public  ISearchCommandsAsync Search => new SearchCommandsAsync(_batch); //search 


    public IDatabaseAsync Db => _batch;
}
```

**Comment by shacharPash** [comment]
> 

The reason I haven't added them until now is because I haven't yet added pipeline tests to each of the modules.
Anyway, in the meantime I've added that for each of the modules 👍

**Review by chayim — APPROVED** [review]
Merge if CI passes

**Pull request merged**